conf/messages.cy

```diff
@@ -95,7 +95,7 @@ hasNirms.change.hidden = Wedi cofrestru ar gyfer y NIRMS
 
 nirmsNumber.title = Beth yw''ch rhif NIRMS?
 nirmsNumber.heading = Beth yw''ch rhif NIRMS?
-nirmsNumber.hint = Mae rhifau NIRMS yn 13 o gymeriadau, ac maent yn dechrau ag RMS, wedyn dash wedi''i ddilyn gan GB neu NI, dash arall, ac yna 6 o rifau. Er enghraifft, RMS-GB-123456.
+nirmsNumber.hint = Mae rhifau NIRMS yn dechrau ag RMS, wedi’i ddilyn gan GB neu NI, ac yna 6 o rifau. Er enghraifft, RMS-GB-123456 neu RMS NI 456789.
```

Comment:
Do you need to add the '' here?

Reply:
No :)

’ is slightly different to '

'' is required for the latter but not the former - they display the same

Reply:
ahh nice :) i've approved